src/InfoViz/Native/FieldSelector/index.js

```diff
@@ -38,8 +38,6 @@ function fieldSelector(publicAPI, model) {
 
     if (el) {
       d3.select(model.container)
-        .style('overflow-y', 'auto')
-        .style('overflow-x', 'hidden');
```

Comment:
Yes it does scroll, the parent need to handle it, but that is the responsibility of the workbench.
You are not supposed to edit the style of your parent and not cleaning it up when you get removed as it will affect other components.

I had to do the same fix on the Histogram selector.

I'll push something to remove L40. Thanks for noticing it...